src/components/Calculator.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+class Calculator extends React.Component {
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/153107463-c198e953-1160-464c-8591-5fdc05680892.png)

Therefore:

- [ ] Please, kindly match the given design in order to meet the project requirements. Your calculator will be beautiful! 💪